recipes/publish.rb

```diff
@@ -29,14 +29,16 @@
 
   # Only build and publish if we have a depot token
   if habitat_depot_token? # ~FC023
-    hab_build node['delivery']['change']['project'] do
-      origin origin
-      plan_dir habitat_plan_dir
-      cwd node['delivery']['workspace']['repo']
-      home_dir delivery_workspace
-      auth_token project_secrets['habitat']['depot_token']
-      depot_url node['habitat-build']['depot-url']
-      action [:build, :publish, :save_application_release]
+    modified_habitat_plan_contexts.each do |plan_context|
```

Comment:
I made a note about this in the README. For more complex scenarios like that, I don't want to try and create a "one size fits all". I'd much rather give them the tools and let them work out the pattern themselves.